Flick/Cells/MediaSeletableCollectionViewCell.swift

```diff
@@ -66,6 +67,13 @@ class MediaSelectableCollectionViewCell: UICollectionViewCell {
         }
     }
 
+    func configure(media: Media) {
```

Comment:
we should eventually make this picture thing an function extension of UIImage haha